modules/jaeger-product-overview.adoc

```diff
@@ -4,7 +4,7 @@ This CONCEPT module included in the following assemblies:
 -rhbjaeger-architecture.adoc
 ////
 
-[id="ossm-jaeger-overview_{context}"]
+[id="jaeger-product-overview_{context}"]
 = Jaeger overview
```

Comment:
Not sure "Jaeger" is required in all the subsections of Jaeger architecture?

Reply:
Did you mean for the overview to be repeated here and in the Jaeger release notes?

Reply:
With the change to modular docs, if you don't start the module names with the component (Jaeger, OSSM) you cannot find your files in the modules folder.